### PR TITLE
Epi autoinjectors are now a subtype of autoinjectors, the basetype is now empty

### PR DESCRIPTION
**Pull request by DGamerL**
## What Does This PR Do
There now an empty variant of the autoinjector now, with the epi becoming a subtype of autoinjectors.

## Why It's Good For The Game
The basetype should be empty as we have a decent amount of autoinjectors already

## Testing
Compiled, the basetype was empty

## Changelog
NPFC

### CODE REVIEW

File: code/modules/reagents/reagent_containers/hypospray.dm [hunk 1]
```diff
@@ -153,6 +152,13 @@
 	else
 		. += "<span class='notice'>It is spent.</span>"
 
+/obj/item/reagent_containers/hypospray/autoinjector/epinephrine
+	name = "emergency autoinjector"
+	desc = "A rapid and safe way to stabilize patients in critical condition for personnel without advanced medical knowledge."
+	icon_state = "autoinjector"
```
Comment (CinnamonSnowball) [inline code/modules/reagents/reagent_containers/hypospray.dm:158]: If the name/description/icon are meant to be the same as its parent type's, it'll just inherit them so you don't need to restate it
```suggestion
```

Comment (DGamerL) [reply code/modules/reagents/reagent_containers/hypospray.dm:158]: Good one, missed this!

File: code/modules/reagents/reagent_containers/hypospray.dm [hunk 2]
```diff
@@ -153,6 +152,10 @@
 	else
 		. += "<span class='notice'>It is spent.</span>"
 
+/obj/item/reagent_containers/hypospray/autoinjector/epinephrine
+	list_reagents = list("epinephrine" = 10)
+	container_type = null // Only epi autoinjectors can't be drained
```
Comment (CinnamonSnowball) [inline code/modules/reagents/reagent_containers/hypospray.dm:157]:
```suggestion
	container_type = NONE // Only epi autoinjectors can't be drained
```
Preferred; even though `null` shouldn't break anything, most other containers use `NONE` so it's good for consistency

File: code/game/objects/items/weapons/storage/bags.dm [hunk 3]
```diff
@@ -531,5 +531,8 @@
 	storage_slots = 25
 	max_combined_w_class = 200
 	w_class = WEIGHT_CLASS_TINY
-	can_hold = list(/obj/item/slime_extract,/obj/item/reagent_containers/food/snacks/monkeycube,/obj/item/reagent_containers/syringe,/obj/item/reagent_containers/glass/beaker,/obj/item/reagent_containers/glass/bottle,/obj/item/reagent_containers/iv_bag,/obj/item/reagent_containers/hypospray/autoinjector)
+	can_hold = list(/obj/item/slime_extract,/obj/item/reagent_containers/food/snacks/monkeycube,
+					/obj/item/reagent_containers/syringe,/obj/item/reagent_containers/glass/beaker,
+					/obj/item/reagent_containers/glass/bottle,/obj/item/reagent_containers/iv_bag,
+					/obj/item/reagent_containers/hypospray/autoinjector/epinephrine)
```
Comment (Henri215) [inline code/game/objects/items/weapons/storage/bags.dm:537]:
```suggestion
	can_hold = list(/obj/item/slime_extract, /obj/item/reagent_containers/food/snacks/monkeycube,
					/obj/item/reagent_containers/syringe, /obj/item/reagent_containers/glass/beaker,
					/obj/item/reagent_containers/glass/bottle, /obj/item/reagent_containers/iv_bag,
					/obj/item/reagent_containers/hypospray/autoinjector/epinephrine)
```

**Review by Henri215 — APPROVED** [review]

File: code/modules/reagents/reagent_containers/hypospray.dm [hunk 4]
```diff
@@ -113,7 +113,7 @@
 	volume = 100
 	list_reagents = list("nanites" = 100)
 
-/obj/item/reagent_containers/hypospray/autoinjector
+/obj/item/reagent_containers/hypospray/autoinjector // This is an empty variant
 	name = "emergency autoinjector"
 	desc = "A rapid and safe way to stabilize patients in critical condition for personnel without advanced medical knowledge."
```
Comment (lewcc) [inline code/modules/reagents/reagent_containers/hypospray.dm:118]: You should probably change the name and description of this to reflect the fact it's empty.

**Review by lewcc — CHANGES_REQUESTED** [review]
One thing. Also, please update the TODO in the PR description if it's been done (which seems to be the case to me)

**Review by lewcc — APPROVED** [review]

**Pull request merged**